share_extension/ios/extension_channels.js

```diff
@@ -112,6 +111,11 @@ export default class ExtensionChannels extends PureComponent {
             const direct = getDirectChannels({entities});
             const channels = channelIds.map((id) => this.props.entities.channels.channels[id]).concat(direct);
 
+            this.publicChannelIcon = await Icon.getImageSource('globe', 16, this.props.theme.centerChannelColor);
```

Comment:
Would it make any noticeable difference if we loaded these in parallel instead of one at a time?

Reply:
Don’t know I can test it